.github/FUNDING.yml

```
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
+
+github: # [suchov] Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
```

```suggestion
github: [suchov] # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
```

Uncommented a list of names.

How does a user know if they are sponsors-enabled? Anyway, I think one or two would be sufficient